providers/implementations/macs/hmac_prov.c

```diff
@@ -59,6 +64,15 @@ struct hmac_data_st {
     int tls_header_set;
     unsigned char tls_mac_out[EVP_MAX_MD_SIZE];
     size_t tls_mac_out_size;
+#ifdef FIPS_MODULE
+    /*
+     * internal is set to 1 if HMAC is used inside another algorithm such as a
```

Comment:
Captial _I_ at the start :grinning:

Reply:
Used 'internal' instead - I want it to match the variable.